docs-beta/src/components/Testimonials.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import "../css/tailwind-styles.css"; 
+
+export const TestimonialList=[
```

Comment:
We should move these "props" objects into the page index.js so that it's centralized (we would update content for these in just that one index.js)

Reply:
Another option would be to pull out the content into a content folder with dedicated js files for each section? I do like the idea of separating the components from the content, but the index is also getting a little long... :)

Reply:
I added a data folder within components that contains 3 JS files exporting the ToolList, FeatureList, and TestimonialList.